AlgebraicPetri-Stratification/src/GrometInterop.jl

```diff
@@ -170,7 +170,7 @@ function semagram2petrinet(sg)
           pn[j_ind, :tname] = Symbol(j["name"])
         end
         if valued
-          pn[j_ind, :rate] = isnothing(j["value"]) ? 0.0 : j["value"]
+          pn[j_ind, :rate] = isnothing(j["value"]) ? 0.0 : parse(Float64, j["value"]["value"]["val"])
```

Comment:
Same comment as with the concentrations, except using `r_type` instead of `c_type`.